sdk/search/azure-search/src/main/java/com/azure/search/models/RangeFacetResult.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.models;
+
+/**
+ * Contains the range facet value found by a search query
+ */
+public class RangeFacetResult {
+    private final Long count;
+    private final Object from;
+    private final Object to;
+
+    /**
+     * Constructor
+     *
+     * @param facetResult facet result object
+     */
+    public RangeFacetResult(FacetResult facetResult) {
+        count = facetResult.getCount();
+        from = facetResult.getDocument().get("from");
+        to = facetResult.getDocument().get("to");
+    }
+
+    /**
+     * Get count
+     *
+     * @return count
+     */
+    public Long getCount() {
+        return count;
+    }
+
+    /**
+     * Get from
+     *
+     * @return from
+     */
+    public Object getFrom() {
+        return from;
+    }
+
+    /**
+     * Get to
+     *
+     * @return to
+     */
+    public Object getTo() {
+        return to;
+    }
```

Comment:
It'd be nice to have slightly more verbose Javadocs to give a deeper explanation about these getters given the property names are so terse.